Datadog.Trace.proj

```diff
@@ -3,6 +3,10 @@
     <BuildInParallel Condition="'$(BuildInParallel)' == ''">true</BuildInParallel>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)'==''">
```

Comment:
I'm pretty sure we don't need this since "Debug" is already the default and you didn't need to add it to other projects.